src/snowflake/snowpark/_internal/compiler/large_query_breakdown.py

```diff
@@ -167,6 +178,10 @@ def _try_to_breakdown_plan(self, root: TreeNode) -> List[LogicalPlan]:
             _logger.debug(
```

Comment:
@sfc-gh-aalam this may not be related to this pr, but i think we can merge the check for skip large query break down into one function, can we do it in a follow up pr?

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-1651211